docs/src/user-guide/getting-started.md

```diff
@@ -25,7 +25,29 @@ You can install and configure ESLint using this command:
 npm init @eslint/config
 ```
 
-**Note:** `npm init @eslint/config` assumes you have a `package.json` file already. If you don't, make sure to run `npm init` or `yarn init` beforehand.
+If you want to use a specific shareable config that is hosted on npm, you can use the `--config` option and specify the package name:
+
+```shell
+# use `eslint-config-semistandard` shared config
+# npm 6.x
+npm init @eslint/config --config semistandard
+
+# ⚠️ npm 7+, extra double-dash is needed:
+npm init @eslint/config -- --config semistandard
+
+# or (`eslint-config` prefix is optional)
+npm init @eslint/config -- --config eslint-config-semistandard
+```
+
+The `--config` flag also supports passing in arrays:
+
+```shell
+npm init @eslint/config -- --config semistandard,standard
+# or
+npm init @eslint/config -- --config semistandard --config standard
+```
+
+**Note:** `npm init @eslint/config` assumes you have a `package.json` file already. If you don't, make sure to run `npm init` beforehand.
```

Comment:
I went with @nzakas idea of not adding all package managers, either yarn or pnpm to the documentation, so only the npm command was kept.

Reply:
This is different from `npm init @eslint/config` which executes `@eslint/create-config` which then asks the user to choose a package manager to install dependencies so it doesn't matter whether it was run through `npm` or another package manager. I wouldn't add more package managers here, but I'd keep yarn init to show that it isn't necessary to run _npm_ init before running `npm init @eslint/config`, and yarn users should indeed run `yarn init` instead of `npm init`.